generator/explores/metric_definitions_explore.py

```diff
@@ -0,0 +1,85 @@
+"""Metric Hub metrics explore type."""
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any, Dict, Iterator, List, Optional
+
+from google.cloud import bigquery
+
+from ..views import View, lookml_utils
+from . import Explore
+
+
+class MetricDefinitionsExplore(Explore):
+    """Metric Hub Metrics Explore."""
+
+    type: str = "metric_definitions_explore"
+
+    def __init__(
+        self,
+        name: str,
+        views: Dict[str, str],
+        views_path: Optional[Path] = None,
+        defn: Optional[Dict[str, Any]] = None,
+    ):
+        """Initialize MetricDefinitionsExplore."""
+        super().__init__(name, views, views_path)
+
+    @staticmethod
+    def from_views(views: List[View]) -> Iterator[Explore]:
+        """Generate an Operational Monitoring explore for this namespace."""
+        for view in views:
+            if view.view_type == "metric_definitions_view":
+                yield MetricDefinitionsExplore(
+                    "metric_definitions",
+                    {"base_view": view.name},
+                )
+
+    @staticmethod
+    def from_dict(name: str, defn: dict, views_path: Path) -> MetricDefinitionsExplore:
+        """Get an instance of this explore from a dictionary definition."""
+        return MetricDefinitionsExplore(name, defn["views"], views_path, defn)
+
+    def _to_lookml(
+        self,
+        bq_client: bigquery.Client,
```

Comment:
It's not used in this particular explore, however we are extending the `Explore` class here and we need to keep the same function parameters for the methods we overridde otherwise `mypy` will complain